functions/internal/runner/handle_tool_call.go

```diff
@@ -157,6 +146,7 @@ func (s *Service) executeRequest(ctx context.Context, req callRequest, w http.Re
 
 	// Announce trailers for resource usage metrics
 	// We currently have to write these after WriteHeader
+	w.Header().Del("Content-Length")
```

Comment:
Why is this necessary if content-length is no longer in the allowedHeaders in the first place